cloudbuild.yaml

```diff
@@ -1,5 +1,21 @@
 steps:
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/cirrus-ci-community/cirrus-ci-agent:$TAG_NAME', '-t', 'gcr.io/cirrus-ci-community/cirrus-ci-agent:latest', '.']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['run', '--privileged', 'linuxkit/binfmt:v0.8']
+    id: 'initialize-qemu'
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['buildx', 'create', '--name', 'mybuilder']
+    id: 'create-builder'
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['buildx', 'use', 'mybuilder']
+    id: 'select-builder'
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['buildx', 'inspect', '--bootstrap']
+    id: 'show-target-build-platforms'
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['buildx', 'build', '--platform', 'linux/amd64,linux/arm64', '-t', 'gcr.io/cirrus-ci-community/cirrus-ci-agent:$TAG_NAME', '-t', 'gcr.io/cirrus-ci-community/cirrus-ci-agent:latest', '.']
+    id: 'build-multi-architecture-container-image'
+options:
+  env:
+    - 'DOCKER_CLI_EXPERIMENTAL=enabled'
```

Comment:
It would be a good idea to comment why we need this variable, because in the future the needed experimental option might move into mainline.

Otherwise LGTM.

Reply:
👍 added in 3333e17b87754e90e3037d01f90fcebd3b39f957. Plus increased the timeout since QEMU will be involved and ARM build should be much slower.